acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/FilecoinJsonRequests.java

```diff
@@ -115,6 +115,10 @@ public static String executeRawJsonRpcRequest(final String url, final String req
     final HttpPost post = new HttpPost(url);
     post.setEntity(new StringEntity(request, Charsets.UTF_8));
     post.setHeader(HttpHeaders.CONTENT_TYPE, MediaType.JSON_UTF_8.toString());
+    final String authToken = System.getenv("ETH2SIGNER_BEARER_TOKEN");
```

Comment:
can we move it up as `private static final Optional<String> authToken = Optional.ofNullable(System.getenv("ETH2SIGNER_BEARER_TOKEN"));`. This will allow any one reading this test class to read it at the beginning instead of buried down here?

Reply:
done